docs/plans/configurable-scoring-system.md

```
@@ -0,0 +1,554 @@
+    dnf: "worst_performance" | "zero" | "last_place"
+    dns: "worst_performance" | "zero" | "exclude"
+    withdrawn: "zero" | "exclude"
+  }
+}
+```
+
+#### Score Multiplier Integration
+
+The existing `trackWorkouts.pointsMultiplier` field is preserved and works with all scoring algorithms. The multiplier applies **after** the scoring algorithm calculates base points:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)